### DISCUSSION TITLE
[Feature] Fool-proof dangerous buttons in UI

### DISCUSSION THREAD

**Discussion by euroborne** [immich-app/immich#14785] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The interface is in need of fool-proofing. I am new to immich and have recently dumped 120k photos into the server. It ran for a few days detecting faces. I just spent hours of trolling through attempting to Hide all the strangers I didn't care to see. Maybe took it too far and my mouse finger became very jittery and eyes twitchy as I was just about to walk away from my task. I moused over the eye button at the top; clicked accidentally before a mouse-over displayed what it was for. One click and all the hours work has gone down the drain. 

Feature: this is a perfect place to add an "are you sure?" confirmation and an Apply function.  One for "Hide unnamed people" and the same for "Reset people visibility". A way of reverting the last change would be nice too. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by ffchung** [comment]
As I think we just need a button to delete the strangers face.

1. Named - important
2. hidden - someone I know but not important
3. delete - strangers

**Comment by euroborne** [comment]
I find immich to provide the most flexible Face management options I have ever seen. I am simply suggesting that the nuke-scoped options for affecting the entire database should be:

- harder to get to by being tucked away under a "global actions" menu
- clearly labeled
- protected by an "are you sure" confirmation
- would be nice to have an "undo" option, yet

Keep in mind photos of events and crowds; like games or tournaments, street, with a potential for thousands of strange faces, where any soft of global reset could destroy hours of manual work.
